README.md

```diff
@@ -1,2 +1,71 @@
 # minikube-local
+
 The best way to set minikube up for local development
+
+[![Build Status](https://circleci.com/gh/tilt-dev/minikube-local/tree/master.svg?style=shield)](https://circleci.com/gh/tilt-dev/minikube-local)
+
+When using Tilt with a [Minikube](https://minikube.sigs.k8s.io/docs/) cluster, 
+we recommend using a local registry for faster image pushing and pulling.
```

Comment:
> faster image pushing and pulling.

seems like you joinked this from a doc where the tradeoff was local registry vs. remote registry -- the main Q for minikube is about local registry vs. hooking into minikube docker, I think the docs should discuss that instead

Reply:
i think you overestimate how well most minikube users are going to understand the minikube feature set. simplified the explanation here a bit